flavors/python-3.6-data-science-cuda-EXASOL-6.2.0/flavor_base/cuda_deps/Dockerfile

```diff
@@ -84,6 +84,7 @@ RUN cd /tmp && \
 ################################################### Install CUDA SDK ###############################################
 
 RUN apt-get update && apt-get install -y --no-install-recommends gnupg curl ca-certificates && \
+    curl -fsSL https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/3bf863cc.pub | apt-key add - && \
```

Comment:
Wouldn't replacing be safer than adding?

Reply:
No, the build break if we keep only the new one.

Reply:
Okay then.